src/gource_shell.cpp

```diff
@@ -197,8 +197,6 @@ Gource* GourceShell::getNext() {
         return 0;
     }
 
-    gGourceSettings.importGourceSettings(*conf, *gource_settings);
```

Comment:
Gource supports config file having mutiple [gource] sections, so you can have multiple configurations in one file and it will cycle to the next one when the current one concludes. So it still needs to handle that, we'll just need to come up with another way to avoid it creating duplicate multi value entries.

Reply:
The overwriting behaviour is kind of nice as it allows for instance having a config file, but also having command line parameters in addition to the config file settings. Appending to existing multi value lists probably isnt useful though.

Reply:
You could avoid dealing with this issue if you used comma separators rather than multi value fields, which would also be quicker to specifiy. e.g. --seconds-per-day 1,2,3 --transition 2020-01-01,2020-01-02